src/TBuffer.h

```diff
@@ -141,6 +141,13 @@ class TBuffer
 
     static const QMap<QString, QVector<QString>> mSupportedMxpElements;
 
+    // TChar is 44bytes on a 64bit system and in testing ends up taking ~60 bytes per character
+    // with all other resources included
+    inline static const int TCHAR_IN_BYTES = 60;
```

Comment:
Why can't we use:
```CPP
    inline static const int TCHAR_IN_BYTES = sizeof(TChar);
```
?

Reply:
Because I didn't know that :grin:  I'll use that and add some on top for the general Mudlet overhead.

Reply:
Well that will already include any padding used to align the members - as the Clang model sometimes seems to point out on some of my systems... :open_mouth:

Reply:
Yes! But Mudlet itself uses memory for more than just storing individual characters - triggers, aliases, etc.

Reply:
Adjusted.